src/hotspot/cpu/aarch64/aarch64_vector.ad

```diff
@@ -447,7 +447,6 @@ instruct storeV(vReg src, vmemA mem) %{
 // vector load/store - predicated
 
 instruct loadV_masked(vReg dst, vmemA mem, pRegGov pg) %{
-  predicate(UseSVE > 0);
```

Comment:
How about

```
  // This predicate is unneeded because only SVE has pRegs.
  // predicate(UseSVE > 0);
```
I still don't like it much, but at least it's clear.

Reply:
> How about
> 
> ```
>   // This predicate is unneeded because only SVE has pRegs.
>   // predicate(UseSVE > 0);
> ```
> 
> I still don't like it much, but at least it's clear.

Hi @theRealAph, thanks for your kind suggestion. I agree on you said. Compared with current simple rules, the changed rules may be not easy to follow. Since code size is not a big problem now, maintaining the status quo seems better.

I'll close the PR. Thanks again for your review!